hasura/migrations/default/1667884678372_alter_table_public_organizations_add_column_created_by/down.sql

```diff
@@ -0,0 +1,4 @@
+-- Could not auto-generate a down migration.
+-- Please write an appropriate down migration for the SQL below:
+-- alter table "public"."organizations" add column "created_by" integer
+--  null;
```

Comment:
no down migration?

Reply:
maybe because of the FK/relationship? I'm not sure. I can add a manual one i guess